src/status/libstatus/wallet.nim

```diff
@@ -84,7 +84,8 @@ proc validateMnemonic*(mnemonic: string): string =
   result = $status_go.validateMnemonic(mnemonic)
 
 proc startWallet*(watchNewBlocks: bool) =
-  discard status_go.startWallet(watchNewBlocks) # TODO: true  to watch trx
+  discard
+  # discard status_go.startWallet(watchNewBlocks) # TODO: true  to watch trx
```

Comment:
This is no longer in status-go. It was a breaking change in the latest status-go. Richard is working on a fix, but it was easier for me for now to just comment it.